Makefile

```diff
@@ -1,5 +1,3 @@
-$(info starting make in kong-build-tools)
-
```

Comment:
@curiositycasualty I want this back. I couldn't find a way to suppress it's output conditionally -- any help appreciated :)